rgw/v2/tests/s3_swift/test_multisite_sync_policy.py

```diff
@@ -96,20 +54,37 @@ def test_exec(config, ssh_con):
                 if config.test_ops["group_create"]:
                     group_status = config.test_ops["group_status"]
                     group_id = "global_group"
-                    group_operation(group_id, "create", group_status)
+                    reusable.group_operation(group_id, "create", group_status)
                     if config.test_ops["flow_create"]:
                         flow_type = config.test_ops["flow_type"]
-                        zone_names = flow_operation(group_id, "create", flow_type)
+                        zone_names = reusable.flow_operation(
+                            group_id, "create", flow_type
+                        )
                     if config.test_ops["pipe_create"]:
-                        pipe_id = pipe_operation(group_id, "create", zone_names)
+                        pipe_id = reusable.pipe_operation(
+                            group_id, "create", zone_names
+                        )
+                    if config.test_ops["group_transition"]:
```

Comment:
just a suggestion,IMO  instead of adding variable with value False in config file where that particular condition is not needed,
we can just include in script level:
if config.test_ops.get("group_transition", False):

Reply:
thanks @ckulal , updated pls review